pkg/exporter/process_test.go

```diff
@@ -31,7 +31,7 @@ func init() {
 
 func TestExportingProcess_SendingTemplateRecordToLocalTCPServer(t *testing.T) {
 	// Create local server for testing
-	listener, err := net.Listen("tcp", ":0")
+	listener, err := net.Listen("tcp", "127.0.0.1:4738")
```

Comment:
what if the port is already in use? `:0` opens a random available port on localhost. What is the issue with this?

Reply:
It will have [G102](https://securego.io/docs/rules/g102.html) error for `gosec` in Golangci-lint. Used another workaround to solve it.